bipolarbridges/companion-app
# Use environment variables and fixed file templates for global project settings

Creates templates and fixed settings per the setup documentation, and a basic environment substitution system for simplifying and consolidating the different settings throughout the project. Once merged, this should prevent us from having any discrepancies in configuration, or from committing any files which contain team-specific values.

**Testing**:
1. Use the steps in the README to initialize a brand new project (should be a quick and easy process)
2. Make sure firebase things (functions, dashboard, etc.) deploy properly.
3. Make sure iOS app builds
4. Make sure Android app builds properly
